bosh_aws_cpi/lib/cloud/aws/helpers.rb

```diff
@@ -4,7 +4,12 @@ module Bosh::AwsCloud
 
   module Helpers
     def default_ephemeral_disk_mapping
-       { '/dev/sdb' => 'ephemeral0' }
+       [
+         {
+           :device_name => '/dev/sdb',
+           :virtual_name => 'ephemeral0',
+         },
+       ]
```

Comment:
The Ruby API for requesting spot requests seemed to only accept it in this format. It's also the closer-to-AWS-API-specs format, so it seemed more correct to change it than try hacky workarounds.